controller/scheduler/fleet.py

```diff
@@ -152,7 +152,7 @@ def _create_container(self, name, image, command, unit, **kwargs):
         tags = kwargs.get('tags', {})
         unit_tags = tags.viewitems()
```

Comment:
Actually, it's used. When the `tags` variable is updated the "view" `unit_tags` is also updated, and that is used at https://github.com/deis/deis/pull/4851/files#diff-43358db3bfa05551b90bcf90c48dfe2cR157